web/controllers/rules/pr/awaiting_review.ex

```diff
@@ -20,6 +20,9 @@ defmodule Dwylbot.Rules.PR.AwaitingReview do
     reviewers = payload["pull_request"]["requested_reviewers"]
     |> Enum.map(&(&1["login"]))
 
+    assignees = get_assignees_login(payload["issue"]["assignees"])
+    allReviewersWereAssigned? = Enum.all?(reviewers, fn(x) -> Enum.member?(assignees, x) end)
```

Comment:
If your using `x` as a function arg, consider just using [function captures](https://elixir-lang.org/crash-course.html#partials-and-function-captures-in-elixir) instead.

```elixir
allReviewersWereAssigned? = Enum.all?(reviewers, &Enum.member?(assignees, &1))
```